_layouts/base.html

```diff
@@ -133,7 +133,7 @@
           </div>
           <div class="grid-col flex-auto">
             <a href="#" class="usa-button usa-button--outline return-to-top-button mobile:display-none desktop:display-block position-fixed bottom-205 right-105 z-500">Return to top</a>
-            <a href="#" class="usa-button usa-button--outline return-to-top-button desktop:display-none position-fixed bottom-205 right-105 width-5 height-5 text-middle radius-pill z-500 shadow-2"><img class="display-inline maxw-3 width-3 height-3 position-absolute left-1 top-1" src="./assets/img/material-icons/expand_less.svg"/></a>
+            <a href="#" class="usa-button usa-button--outline return-to-top-button desktop:display-none position-fixed bottom-205 right-105 width-5 height-5 text-middle radius-pill z-500 shadow-2"><img class="display-inline maxw-3 width-3 height-3 position-absolute left-1 top-1" src="/assets/img/material-icons/expand_less.svg"/></a>
```

Comment:
Can you include `{{ site.baseurl }}` so this works correctly in preview sites. Currently it's broken there.

https://federalist-14ccf384-8bd5-4e30-aa95-a5fe5894e3c1.sites.pages.cloud.gov/preview/18f/identity-dev-docs/jcurcio/9739-return-to-top-enhancement/

![image](https://github.com/18F/identity-dev-docs/assets/1779930/c82d8d44-c2f9-447a-bd25-2b70a6ee3f96)

```suggestion
            <a href="#" class="usa-button usa-button--outline return-to-top-button desktop:display-none position-fixed bottom-205 right-105 width-5 height-5 text-middle radius-pill z-500 shadow-2"><img class="display-inline maxw-3 width-3 height-3 position-absolute left-1 top-1" src="{{ site.baseurl }}/assets/img/material-icons/expand_less.svg"/></a>
```

Reply:
Perfect! Thank you Andrew. I knew there must be a way to make it work in Federalist, Local, and Production. But didn't think of using mustache!